tests/test-sessions/test-gcloud-sessions/src/test/java/org/eclipse/jetty/gcloud/session/GCloudSessionTestSupport.java

```diff
@@ -130,15 +130,15 @@ public void createSession (String id, String contextPath, String vhost,
                                       long lastAccessed, long maxIdle, long expiry,
                                       long cookieset, long lastSaved,
                                       Map<String,Object> attributes)
-    throws Exception
+                                              throws Exception
     {
         //serialize the attribute map
         ByteArrayOutputStream baos = new ByteArrayOutputStream();
         if (attributes != null)
         {
+            SessionData tmp = new SessionData(id, contextPath, vhost, created, accessed, lastAccessed, maxIdle);
             ObjectOutputStream oos = new ObjectOutputStream(baos);
-            oos.writeObject(attributes);
-            oos.flush();
+            SessionData.serializeAttributes(tmp, oos);
```

Comment:
`ObjectOutputStream` not closed, use try-with-resources.

Reply:
ok